data/endpoints/fps-initiate-payment-v3.json

```diff
@@ -1,777 +1,816 @@
+          },
+          "BatchPaymentInstructionCounterpartAccountIdentification": {
+              "type": "object",
+              "properties": {
+                  "iban": {
+                      "type": "string",
+                      "description": "The International Bank Account Number of the account.",
+                      "nullable": true
+                  },
+                  "other": {
+                      "$ref": "#/components/schemas/BatchCounterpartAccountGenericIdentification",
+                      "description": "Unique identification of an account, as assigned by the account servicer, using an identification scheme.",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "The identifiable information of an account."
+          },
+          "BatchRemittanceInformation": {
+              "type": "object",
+              "properties": {
+                  "structured": {
+                      "$ref": "#/components/schemas/BatchStructured",
+                      "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (e.g., a purchase reference number).",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information supplied to enable the matching/reconciliation of an entry with the items that the payment is intended to settle, such as commercial invoices in an accounts’ receivable system."
+          },
+          "BatchStructured": {
+              "type": "object",
+              "properties": {
+                  "creditorReferenceInformation": {
+                      "$ref": "#/components/schemas/BatchCreditorReferenceInformation",
+                      "description": "Reference information provided by the ultimate debtor to allow the identification of underlying documents by the creditor.",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (e.g., a purchase reference number)."
+          },
+          "CounterpartAccountGenericIdentificationScheme": {
+              "type": "object",
+              "properties": {
+                  "code": {
+                      "enum": [
+                          "BBAN",
+                          "CUID",
+                          "UPIC"
+                      ],
+                      "type": "string",
+                      "description": "Name of the identification scheme in coded form. No need to specify if you are providing this information in free-form text (i.e., in the \"proprietary\" field).",

```

Comment:
Same idea with i.e. I have no problems with it, but some style guides don't like Latinisms. (_that is_, in the \"proprietary\" field)

Reply:
```suggestion
                      "description": "Name of the identification scheme in coded form. No need to specify if you are providing this information in free-form text (in the \"proprietary\" field).",

```

Reply:
```suggestion
                      "description": "Name of the identification scheme in coded form. No need to specify if you are providing this information in free-form text in the \"proprietary\" field.",

```